content/en/service_management/case_management/create_case.md

```diff
@@ -36,16 +41,29 @@ You can also create cases manually from the following products:
 ## Automatic case creation
 
 Configure the following products to automatically create cases:
-| Product | Instructions    | 
-| ------  | ----------- | 
+| Product | Instructions    |
+| ------  | ----------- |
 | Monitors | Navigate to the [Project Settings page][4], click **Integrations** > **Datadog Monitors**, and click on the toggle to get your @case-<project_handle>. <br><br> When creating a monitor, include `@case-{project_handle}` in the **Configure notifications and automations** section. Cases are automatically created when the monitor transitions to a different status. To only create cases for certain monitor transitions, use [conditional variables][3]. As an example, to create cases only when a monitor triggers, wrap the `@case` mention with `{{#is_alert}}` and `{{/is_alert}}`.<br><br> Toggle on **Auto-close cases when the monitor group resolves** to reduce manual cleanup.|
 | Event Management (Correlations) | In Event Management, correlations configured to aggregate events from Datadog and third-party sources automatically create cases.   |
 | Workflow Automation | 1. In a new or existing workflow, add a step in the Workflow builder and search for "Case Management."<br> 2. Select the **Create Case** action.<br> 3. If the workflow is configured to run based on a monitor or security signal trigger, add the relevant workflow triggers and ensure that you've added the workflow handle to the desired resources. For more information, see [Trigger a workflow][6].|
 | Error Tracking | In Error Tracking, cases are automatically created when an issue is commented on or assigned. |
 
+## Case types
+
+Add case types when you are creating a case. Not all case types are available for configuration between manual and automativ creation. For example, only `Standard`, `Security` and `Change Request`, `Event Management` types are available when creating cases manually.
```

Comment:
```suggestion
Add case types when you are creating a case. Not all case types are available for configuration between manual and automativ creation. For example, only `Standard`, `Security` and `Change Request`, `Event Management` types are available when creating cases manually.

To add and enable custom case types, see [Case Customization][7].
```

Reply:
looks like this line 55 still needs committed just a reminder to do so :)